### PR TITLE
Add sound-juicer

### PR DESCRIPTION
**Pull request by hadess**
(yes, I keep adding my build scripts because my brain is small, and I'll do it when you're not looking anyway)

### CODE REVIEW

**Comment by hadess** [comment]
bot, build org.gnome.SoundJuicer

File: org.gnome.SoundJuicer.json [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+        /* Migrate DConf settings from the host */
+        "--metadata=X-DConf=migrate-path=/org/gnome/sound-juicer/",
+        /* optical media detection */
+        "--talk-name=org.gtk.vfs", "--talk-name=org.gtk.vfs.*",
+        /* Force volume monitor with optical media detection support */
+        "--env=GIO_USE_VOLUME_MONITOR=udisks2",
+        /* Ensure cdda gstreamer plugin is picked found for audio CD's */
+        "--env=GST_PLUGIN_PATH=/app/lib/codecs/lib/gstreamer-1.0"
+    ],
+    "build-options" : {
```
Comment (nedrichards) [inline org.gnome.SoundJuicer.json:29]: these should be unnecessary in 3.36 (as in - they're already set in the SDK)

**Comment by nedrichards** [comment]
Other than OARS and release tags looks initially decent. Thanks for adding this, as it provides the invaluable opportunity to troll @rossburton about https://gitlab.gnome.org/GNOME/sound-juicer/-/blob/master/MAINTAINERS

**Comment by hadess** [comment]
bot, build org.gnome.SoundJuicer

**Comment by hadess** [comment]
bot, build org.gnome.SoundJuicer

**Comment by hadess** [comment]
Not sure whether https://github.com/flatpak/flatpak/pull/3839 should be a blocker.

I've fixed all the warnings I could see during a normal run, and all the profiles listed are supported out-of-the-box, so ready to go for me.

**Comment by nedrichards** [comment]
bot, build org.gnome.SoundJuicer

**Comment by nedrichards** [comment]
I can't seem to actually make this see my USB CD drive, which is unfortunate. The dconf migrate issue isn't a blocker though I think.

**Comment by hadess** [comment]
> I can't seem to actually make this see my USB CD drive, which is unfortunate. The dconf migrate issue isn't a blocker though I think.

Are you running this under GNOME? What's the output of `gio mount -li` from inside the sandbox?

**Comment by nedrichards** [comment]
gio sees it - further testing on other devices suggests that this is a mechanical issue with the drive as much as anything else. Sorry for the false alarm.

**Pull request closed without merging**